apps/condo/domains/ticket/schema/Ticket.test.js

```diff
@@ -1428,4 +1436,848 @@ describe('Ticket', () => {
         })
 
     })
+    describe( 'NotificationWhen', () => {
+        // it('update status to TICKET_STATUS_OPENED and send push for resident with registered pushToken', async () => {
```

Comment:
why it's commented?

Reply:
This is commented out due to the customer field being empty when the ticket was created. When this changes, the tests should work